Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4335,16 +4640,21 @@ class AttackPattern(Indicator):
         """
         CONTEXT_PATH = 'AttackPattern(val.value && val.value == obj.value)'
 
-        def __init__(self, stix_id, kill_chain_phases, first_seen_by_source, description,
-                     operating_system_refs, publications, mitre_id, tags, dbot_score):
-            self.stix_id = stix_id
-            self.kill_chain_phases = kill_chain_phases
-            self.first_seen_by_source = first_seen_by_source
+        def __init__(self, stix_id, kill_chain_phases=None, first_seen_by_source=None, description=None,
+                     operating_system_refs=None, publications=None, mitre_id=None, tags=None,
+                     traffic_light_protocol=None, dbot_score=None, community_notes=None, external_references=None):
```

Comment:
Please add the new fields to the class's docstring (describe their type and description).

Reply:
Done